core/src/cg/group4/game_logic/StandUp.java

```diff
@@ -31,11 +32,6 @@
     protected static StandUp cInstance;
 
     /**
-     * Background music to be played.
-     */
-    protected Music cBackgroundMusic;
```

Comment:
Not something for this branch, but has to be done. I'd suggest creating an Issue and assigning it to the person that created it.